lib/hanami/logger.rb

```diff
@@ -156,7 +157,7 @@ class JSONFormatter < Formatter
       # @since x.x.x
       # @api private
       def _format(hash)
-        JSON.generate(hash)
+        Hanami::Utils::Json.dump(hash)
```

Comment:
I think no. I want to use hanami json wrapper here, that's why I use downcased `json` keyword